ProcessMaker/Notifications/ErrorExecutionNotification.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+namespace ProcessMaker\Notifications;
+
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Notifications\Messages\BroadcastMessage;
+use Illuminate\Notifications\Messages\MailMessage;
+use Illuminate\Notifications\Notification;
+use ProcessMaker\Nayra\Contracts\Bpmn\TokenInterface;
+
+class ErrorExecutionNotification extends Notification
+{
+    use Queueable;
+
+    private $tokenInterface;
+    private $message;
+    private $errorHandling;
+
+    /**
+     * Create a new notification instance.
+     *
+     * @return void
+     */
+    public function __construct(TokenInterface $tokenInterface, $message = '', $errorHandling = [])
+    {
+        $this->tokenInterface = $tokenInterface;
+        $this->message = $message;
+        $this->errorHandling = $errorHandling;
+    }
+
+    /**
+     * Get the notification's delivery channels.
+     *
+     * @param  mixed  $notifiable
+     * @return array
+     */
+    public function via($notifiable)
+    {
+        $via = [];
+        if ($this->errorHandling['inapp_notification'] === true) {
+            $via[] = NotificationChannel::class;
```

Comment:
I think this also needs 'broadcast' so it sends the bell icon in real-time. I added some tests you can run that should pass when broadcast is there: tests/Feature/Processes/ErrorExecutionNotificationTest.php